test/behaviour/typeql/TypeQLSteps.java

```diff
@@ -159,6 +169,31 @@ public void typeql_match(String typeQLQueryStatements) {
         }
     }
 
+    private void testProcedurePermutations(TypeQLMatch typeQLQuery) {
```

Comment:
This is an assertion method. Therefore, we should follow the naming convention that is applicable.